theme/static/js/resourceLandingAjax.js

```diff
@@ -1011,33 +1109,52 @@ function initializeDatePickers(){
 }
 
 // act on spatial coverage type change
-function setFileTypeSpatialCoverageFormFields(){
-    $("#div_id_type_filetype input:radio").change(function () {
-        if ($(this).val() == "point" && $(this).attr("checked") == "checked") {
-            $("#div_id_north_filetype").show();
-            $("#div_id_east_filetype").show();
-            $("#div_id_elevation_filetype").show();
-            $("#div_id_northlimit_filetype").hide();
-            $("#div_id_eastlimit_filetype").hide();
-            $("#div_id_southlimit_filetype").hide();
-            $("#div_id_westlimit_filetype").hide();
-            $("#div_id_uplimit_filetype").hide();
-            $("#div_id_downlimit_filetype").hide();
+function setFileTypeSpatialCoverageFormFields(logical_type){
```

Comment:
@alvacouch reification is not necessarily a bad idea.  It does add complexity and possibly clutter to the XML files.  It is broader than just this netCDF file issue, so I suggest this PR go ahead and that you pull this suggestion out into a new feature request/enhancement to consider in the future.